src/matchers/utils.ts

```diff
@@ -113,4 +113,25 @@ export const getElementText = async (
   throw new Error(`Invalid input length: ${args.length}`)
 }
 
-export const quote = (val: string | null) => `'${val}'`
+export const quote = (val: string | null) => (val === null ? "" : `'${val}'`)
+
+export const getMessage = (
+  { isNot, promise, utils }: jest.MatcherContext,
+  matcher: string,
+  expected: string | null,
+  received: string | null
+) => {
+  const matcherHint =
+    utils.matcherHint(matcher, undefined, undefined, { isNot, promise }) +
+    "\n\n"
+
+  if (isNot) {
+    return matcherHint + `Expected: not ${utils.printExpected(expected)}`
+  } else {
+    return (
+      matcherHint +
+      `Expected: ${utils.printExpected(expected)}\n` +
+      `Received: ${utils.printReceived(received)}`
+    )
+  }
```

Comment:
Maybe it will be better?
```js
const message = isNot ? `Expected: not ${utils.printExpected(expected)}` :
  `Expected: ${utils.printExpected(expected)}\n` +
  `Received: ${utils.printReceived(received)}`

return matcherHint + message
```

Reply:
Good thought, updated.